projects/ti_to_ti/README.md

```diff
@@ -0,0 +1,159 @@
+# Chameleon Model Doc for Align-Anything
+
+## Environment setup
+
+Currently, the official Transformer repo does not support Chameleon model with image output (see [this PR](https://github.com/huggingface/transformers/pull/32013) for more details), so we rely on a certain fork of the repo.
+
+After installing Align-Anything and correctly set up the envrionment, you can install the forked stable version of the repo by running:
+
+```bash
+pip install git+https://github.com/htlou/transformers.git@hantao_stable_cham
+```
+
+## Model Download
+
+We will update this section after our model is uploaded to the Hugging Face model hub.
+
+## Model Training
+
+As chameleon expands the \<image\> in the input into image placeholders (\<image\> * 1024) in the processor, but add the real image tokens inside the forward function, we need to do pretokenize to the input and merge the real image tokens to the labels before doing the training.
+
+### Pre Tokenization
+
+Set the input, output and Chameleon model path correctly in `pre_tokenize_example.py` and run:
+
+```bash
+python pre_tokenize_example.py
+```
+
+If you are dealing with a large dataset, you can use `pre_tokenize_parallel_example.py` to speed up the process in multiple threads:
+
+```bash
+python pre_tokenize_parallel_example.py
+```
+
+### Chameleon Finetuning
```

Comment:
Change to `Model Finetuning`

Reply:
fixed in https://github.com/PKU-Alignment/align-anything/pull/41/commits/068a8bbbaf4a27c191a98f19007a7f354d922d7b